pipeline-runner/R/gem2s-5-create_seurat.R

```diff
@@ -70,20 +74,20 @@ construct_scdata <- function(counts, doublet_score, edrops_out, sample, annot, c
 # construct metadata for each SeuratObject
 construct_metadata <- function(counts, sample, config) {
   message("Constructing metadata df...")
```

Comment:
what is `df`?

Reply:
stands for dataframe.
```suggestion
  message("Constructing metadata table...")
```